### DISCUSSION TITLE
react build static site no route in  repo name, react-router-dom error? github-pages error

### DISCUSSION THREAD

**Discussion by jozhuare99** [community/community#68906] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

site
https://jozhuare99.github.io/kook/

error is no router for "/<reponame>",
running in local machine also have an error like this,
if i browse to localhost:3000/index.html

but works fine if index.html is removed

localhost:3000/

**Comment by jozhuare99** [comment]
I'm sure that I set the naming properly for the link of js and CSS file on the index.html but why it only serve index.html?

**Comment by radar-blip** [comment]
In your index.html, you dont need the slash in your link to css
Just change this link: 
`<link rel="stylesheet" href="/index-e313ae00.css">`

to this:
`<link rel="stylesheet" href="index-e313ae00.css">`

Same with the link to js

**Reply by jozhuare99** [reply]
github pages info page didnt say about that, i will try again i hope it fix

**Reply by radar-blip** [reply]
It should work, but yes, try again.

**Reply by jozhuare99** [reply]
it now comes with an error in my js file :204 No routes matched location "/kook/".

**Reply by radar-blip** [reply]
Try this for js:
`<script type="module" crossorigin src="./index-e8900fb1.js">`

**Reply by jozhuare99** [reply]
![image](https://github.com/community/community/assets/47843347/8eca5041-5384-449e-8f9c-1e97a13372b9)
its still has an error, i guess theres no way to fix this

**Reply by radar-blip** [reply]
There may be a way to fix it, I'm not sure. Someone else might come along who knows the fix.
Also, feel free to unmark my answer since it didn't help.

**Reply by jozhuare99** [reply]
I'm so frustrated the way this GitHub pages work I only want to serve my react project. for my portfolio,

**Reply by radar-blip** [reply]
I'm looking at your code again, I'm not that familiar with how javascript works but
what is the crossorigin for? are you sure you need that in there?

`<script type="module" crossorigin src="./assets/index-0489078c.js"></script>`

**Comment by seyLu** [comment]
#### Hi @jozhuare99,

I took a look at your site. Your site is properly deployed, but it is borderline impossible to debug when js file is uglified and when we don't know the react source code. 
![image](https://github.com/community/community/assets/98249191/31cd1a94-c407-4591-8c5f-2784fd8fc175)
![image](https://github.com/community/community/assets/98249191/5148e15b-4ad7-4d1d-ba25-a9eb44c1e477)
![image](https://github.com/community/community/assets/98249191/ca7ecb4b-f9a2-452d-b366-7fe804599934)

Other routes/files are found just fine, albeit the weird jquery outside the script tag:
![image](https://github.com/community/community/assets/98249191/04a84b23-7f74-42d6-841f-b4dfe6ad436d)
![image](https://github.com/community/community/assets/98249191/b70c4ba1-536c-4386-9884-67d652229ab6)

TLDR:
- show the react source code and we debug from there

**Reply by jozhuare99** [reply]
dont worry about it the index file dont link to that file, index file is the only im concerned of, 

serving the page on local machine also have an issue, it doesn't recognized the directory if it browse in the localhost:3000/index.html ,
ive got the same error as the github page got

 but when i removed the index.html, 
 
 localhost:3000/ my site works fine,
